react-app/src/components/table/DataTable.tsx

```diff
@@ -476,7 +476,13 @@ export const FilterSearchDataGrid = (props: FilterSearchDataGridProps) => {
   // Sets quickfilter value of DataGrid. newValue is a string input.
   const updateSearchValue = useMemo(() => {
     return debounce((newValue) => {
-      tableApiRef.current.setQuickFilterValues(newValue.split(' ').filter((word) => word !== ''));
+      tableApiRef.current.setQuickFilterValues(
+        newValue
+          .replace(/^0+/, '')
+          .replaceAll('-', '')
+          .split(' ')
+          .filter((word) => word !== ''),
+      );
```

Comment:
This solution would affect every search on every page being done with the keyword search bar. 

I would suggest instead looking at the `propertiesServices.ts` file in the backend. Currently on line 586, it adds the quickfilter options. If the foreach loop on `quickFilterFields` pushed `constructFindOptionFromQueryPid` instead for the PID and PIN fields, I think this would solve the issue more specifically.

Reply:
okay i tried to implement this as you suggested. Still open to any and all feedback. Also would we want this for PID and PIN? are pins formatted the same as pids?